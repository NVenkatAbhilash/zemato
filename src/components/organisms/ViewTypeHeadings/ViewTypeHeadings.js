import React from "react";
import { makeStyles } from "@material-ui/core/styles";
import { Grid } from "@material-ui/core";
import Typography from "../../atoms/Typography/Typography";
import Collections from "../../templates/Collections/Collections";
import { Link } from "react-router-dom";

const useStyles = makeStyles({
  root: {
    padding: "0 40px",
  },
  item: {
    margin: "20px 0",
  },
  subHeading: {
    color: "rgb(100,100,100,.9)",
    cursor: "pointer",
  },
  active: {
    color: "rgb(240,0,0,.7)",
    borderBottom: "3px solid rgb(240,0,0,.5)",
  },
  subHeadingBar: {
    padding: "0 0 60px 36px",
  },
  link: {
    textDecoration: "none",
    color: "inherit",
  },
});

const viewTypeList = [
  {
    name: "Delivery",
    heading: "Delivery",
    link: "delivery",
  },
  {
    name: "Dining Out",
    heading: "Dine-Out",
    link: "dineOut",
  },
  {
    name: "Nightlife",
    heading: "Nightlife",
    link: "nightLife",
  },
];

const ViewTypeHeadings = ({active, ...props}) => {
  const classes = useStyles();

  let activeType = 0;
  if (active) {
    activeType = active;
  }

  return (
    <Grid container direction="column" className={classes.root} {...props}>
      <Grid
        item
        direction="row"
        container
        spacing={8}
        className={classes.subHeadingBar}
      >
        {viewTypeList.map((viewTypeItem, index) => {
          let classList = classes.subHeading;
          if (index === activeType) {
            classList += " " + classes.active;
          }
          return (
            <Grid item className={classList} key={index}>
              <Link
                to={"/viewType/" + viewTypeItem.link}
                className={classes.link}
              >
                <Typography variant="h5">{viewTypeItem.name}</Typography>
              </Link>
            </Grid>
          );
        })}
      </Grid>
      {activeType > 0 && (
        <Grid item className={classes.greyBackground}>
          <Collections />
        </Grid>
      )}
      <Grid item>
        <Typography variant="h4">
          {viewTypeList[activeType].heading} Restaurants in Hyderabad
        </Typography>
      </Grid>
    </Grid>
  );
};

export default ViewTypeHeadings;
