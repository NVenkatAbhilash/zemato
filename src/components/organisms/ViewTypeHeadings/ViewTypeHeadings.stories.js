import React from "react";
import ViewTypeHeadings from "./ViewTypeHeadings";

export default {
    title: "organisms/ViewTypeHeadings",
    component: ViewTypeHeadings,
}


const Template = ({...args})=>(
    <ViewTypeHeadings {...args} />
);


export const Delivery = Template.bind({});
Delivery.args = {
    active: 0,
}

export const DineOut = Template.bind({});
DineOut.args = {
    active: 1,
}

export const Nightlife = Template.bind({});
Nightlife.args = {
    active: 2,
}
