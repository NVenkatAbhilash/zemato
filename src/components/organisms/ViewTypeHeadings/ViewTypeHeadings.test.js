import React from "react";
import ReactDOM from "react-dom";
import { BrowserRouter } from "react-router-dom";
import "@testing-library/jest-dom";
import ViewTypeHeadings from "./ViewTypeHeadings";
import renderer from "react-test-renderer";

it("renders ViewTypeHeadings snapshot correctly 1", () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <ViewTypeHeadings />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders ViewTypeHeadings snapshot correctly 2", () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <ViewTypeHeadings active="1" />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders ViewTypeHeadings snapshot correctly 3", () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <ViewTypeHeadings active="2" />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
it("renders ViewTypeHeadings without crashing 1", () => {
  const div = document.createElement("div");
  ReactDOM.render(
    <BrowserRouter>
      <ViewTypeHeadings />
    </BrowserRouter>,
    div
  );
});

it("renders ViewTypeHeadings without crashing 2", () => {
  const div = document.createElement("div");
  ReactDOM.render(
    <BrowserRouter>
      <ViewTypeHeadings active="2" />
    </BrowserRouter>,
    div
  );
});
