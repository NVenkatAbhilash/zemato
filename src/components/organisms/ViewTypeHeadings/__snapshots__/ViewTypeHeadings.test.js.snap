// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ViewTypeHeadings snapshot correctly 1 1`] = `
<div
  className="MuiGrid-root makeStyles-root-1 MuiGrid-container MuiGrid-direction-xs-column"
>
  <div
    className="MuiGrid-root makeStyles-subHeadingBar-5 MuiGrid-container MuiGrid-spacing-xs-8 MuiGrid-item"
  >
    <div
      className="MuiGrid-root makeStyles-subHeading-3 makeStyles-active-4 MuiGrid-item"
    >
      <a
        className="makeStyles-link-6"
        href="/viewType/delivery"
        onClick={[Function]}
      >
        <h5
          className="MuiTypography-root MuiTypography-h5"
        >
          Delivery
        </h5>
      </a>
    </div>
    <div
      className="MuiGrid-root makeStyles-subHeading-3 MuiGrid-item"
    >
      <a
        className="makeStyles-link-6"
        href="/viewType/dineOut"
        onClick={[Function]}
      >
        <h5
          className="MuiTypography-root MuiTypography-h5"
        >
          Dining Out
        </h5>
      </a>
    </div>
    <div
      className="MuiGrid-root makeStyles-subHeading-3 MuiGrid-item"
    >
      <a
        className="makeStyles-link-6"
        href="/viewType/nightLife"
        onClick={[Function]}
      >
        <h5
          className="MuiTypography-root MuiTypography-h5"
        >
          Nightlife
        </h5>
      </a>
    </div>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item"
  >
    <h4
      className="MuiTypography-root MuiTypography-h4"
    >
      Delivery
       Restaurants in Hyderabad
    </h4>
  </div>
</div>
`;

exports[`renders ViewTypeHeadings snapshot correctly 2 1`] = `
<div
  className="MuiGrid-root makeStyles-root-1 MuiGrid-container MuiGrid-direction-xs-column"
>
  <div
    className="MuiGrid-root makeStyles-subHeadingBar-5 MuiGrid-container MuiGrid-spacing-xs-8 MuiGrid-item"
  >
    <div
      className="MuiGrid-root makeStyles-subHeading-3 MuiGrid-item"
    >
      <a
        className="makeStyles-link-6"
        href="/viewType/delivery"
        onClick={[Function]}
      >
        <h5
          className="MuiTypography-root MuiTypography-h5"
        >
          Delivery
        </h5>
      </a>
    </div>
    <div
      className="MuiGrid-root makeStyles-subHeading-3 MuiGrid-item"
    >
      <a
        className="makeStyles-link-6"
        href="/viewType/dineOut"
        onClick={[Function]}
      >
        <h5
          className="MuiTypography-root MuiTypography-h5"
        >
          Dining Out
        </h5>
      </a>
    </div>
    <div
      className="MuiGrid-root makeStyles-subHeading-3 MuiGrid-item"
    >
      <a
        className="makeStyles-link-6"
        href="/viewType/nightLife"
        onClick={[Function]}
      >
        <h5
          className="MuiTypography-root MuiTypography-h5"
        >
          Nightlife
        </h5>
      </a>
    </div>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item"
  >
    <div
      className="MuiGrid-root makeStyles-root-8 MuiGrid-container MuiGrid-direction-xs-column"
    >
      <div
        className="MuiGrid-root MuiGrid-item"
      >
        <h4
          className="MuiTypography-root MuiTypography-h4"
        >
          Collections
        </h4>
      </div>
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-item"
      >
        <div
          className="MuiGrid-root makeStyles-subHeading-9 MuiGrid-item MuiGrid-grid-lg-true"
        >
          <h6
            className="MuiTypography-root MuiTypography-subtitle1"
          >
            Explore curated lists of top restaurants, cafes, pubs, and bars in Hyderabad, based on trends
          </h6>
        </div>
        <div
          className="MuiGrid-root makeStyles-link-10 MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-flex-end MuiGrid-justify-xs-flex-end MuiGrid-grid-lg-3"
        >
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <h6
              className="MuiTypography-root makeStyles-link-7 MuiTypography-subtitle1"
              mode="link"
            >
              All Collections in Hyderabad
            </h6>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M10 17l5-5-5-5v10z"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item"
      >
        <div
          className="MuiGrid-root makeStyles-root-11 MuiGrid-container MuiGrid-spacing-xs-4"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3"
          >
            <a
              href="/collections"
              onClick={[Function]}
              style={
                Object {
                  "textDecoration": "none",
                }
              }
            >
              <div
                className="MuiGrid-root makeStyles-root-12 MuiGrid-container MuiGrid-align-items-xs-flex-end"
                data-testid="main"
                style={
                  Object {
                    "backgroundImage": "linear-gradient(rgba(0,0,0, 0.1), rgb(0, 0, 0, .6)), url(collection1.jpg)",
                  }
                }
              >
                <div
                  className="MuiGrid-root makeStyles-content-14 MuiGrid-item"
                >
                  <h6
                    className="MuiTypography-root MuiTypography-h6"
                    data-testid="special"
                  >
                    New Year's Specials
                  </h6>
                  <h6
                    className="MuiTypography-root MuiTypography-subtitle2"
                    data-testid="place"
                  >
                    12
                     Places
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root makeStyles-arrowIcon-15 MuiSvgIcon-fontSizeSmall"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M10 17l5-5-5-5v10z"
                      />
                    </svg>
                  </h6>
                </div>
              </div>
            </a>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3"
          >
            <a
              href="/collections"
              onClick={[Function]}
              style={
                Object {
                  "textDecoration": "none",
                }
              }
            >
              <div
                className="MuiGrid-root makeStyles-root-12 MuiGrid-container MuiGrid-align-items-xs-flex-end"
                data-testid="main"
                style={
                  Object {
                    "backgroundImage": "linear-gradient(rgba(0,0,0, 0.1), rgb(0, 0, 0, .6)), url(collection2.jpg)",
                  }
                }
              >
                <div
                  className="MuiGrid-root makeStyles-content-14 MuiGrid-item"
                >
                  <h6
                    className="MuiTypography-root MuiTypography-h6"
                    data-testid="special"
                  >
                    Trending This Week
                  </h6>
                  <h6
                    className="MuiTypography-root MuiTypography-subtitle2"
                    data-testid="place"
                  >
                    31
                     Places
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root makeStyles-arrowIcon-15 MuiSvgIcon-fontSizeSmall"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M10 17l5-5-5-5v10z"
                      />
                    </svg>
                  </h6>
                </div>
              </div>
            </a>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3"
          >
            <a
              href="/collections"
              onClick={[Function]}
              style={
                Object {
                  "textDecoration": "none",
                }
              }
            >
              <div
                className="MuiGrid-root makeStyles-root-12 MuiGrid-container MuiGrid-align-items-xs-flex-end"
                data-testid="main"
                style={
                  Object {
                    "backgroundImage": "linear-gradient(rgba(0,0,0, 0.1), rgb(0, 0, 0, .6)), url(collection3.jpg)",
                  }
                }
              >
                <div
                  className="MuiGrid-root makeStyles-content-14 MuiGrid-item"
                >
                  <h6
                    className="MuiTypography-root MuiTypography-h6"
                    data-testid="special"
                  >
                    Hyderabad's Hottest
                  </h6>
                  <h6
                    className="MuiTypography-root MuiTypography-subtitle2"
                    data-testid="place"
                  >
                    111
                     Places
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root makeStyles-arrowIcon-15 MuiSvgIcon-fontSizeSmall"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M10 17l5-5-5-5v10z"
                      />
                    </svg>
                  </h6>
                </div>
              </div>
            </a>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3"
          >
            <a
              href="/collections"
              onClick={[Function]}
              style={
                Object {
                  "textDecoration": "none",
                }
              }
            >
              <div
                className="MuiGrid-root makeStyles-root-12 MuiGrid-container MuiGrid-align-items-xs-flex-end"
                data-testid="main"
                style={
                  Object {
                    "backgroundImage": "linear-gradient(rgba(0,0,0, 0.1), rgb(0, 0, 0, .6)), url(collection4.jpg)",
                  }
                }
              >
                <div
                  className="MuiGrid-root makeStyles-content-14 MuiGrid-item"
                >
                  <h6
                    className="MuiTypography-root MuiTypography-h6"
                    data-testid="special"
                  >
                    Work Friendly Places!
                  </h6>
                  <h6
                    className="MuiTypography-root MuiTypography-subtitle2"
                    data-testid="place"
                  >
                    221
                     Places
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root makeStyles-arrowIcon-15 MuiSvgIcon-fontSizeSmall"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M10 17l5-5-5-5v10z"
                      />
                    </svg>
                  </h6>
                </div>
              </div>
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item"
  >
    <h4
      className="MuiTypography-root MuiTypography-h4"
    >
      Dine-Out
       Restaurants in Hyderabad
    </h4>
  </div>
</div>
`;

exports[`renders ViewTypeHeadings snapshot correctly 3 1`] = `
<div
  className="MuiGrid-root makeStyles-root-1 MuiGrid-container MuiGrid-direction-xs-column"
>
  <div
    className="MuiGrid-root makeStyles-subHeadingBar-5 MuiGrid-container MuiGrid-spacing-xs-8 MuiGrid-item"
  >
    <div
      className="MuiGrid-root makeStyles-subHeading-3 MuiGrid-item"
    >
      <a
        className="makeStyles-link-6"
        href="/viewType/delivery"
        onClick={[Function]}
      >
        <h5
          className="MuiTypography-root MuiTypography-h5"
        >
          Delivery
        </h5>
      </a>
    </div>
    <div
      className="MuiGrid-root makeStyles-subHeading-3 MuiGrid-item"
    >
      <a
        className="makeStyles-link-6"
        href="/viewType/dineOut"
        onClick={[Function]}
      >
        <h5
          className="MuiTypography-root MuiTypography-h5"
        >
          Dining Out
        </h5>
      </a>
    </div>
    <div
      className="MuiGrid-root makeStyles-subHeading-3 MuiGrid-item"
    >
      <a
        className="makeStyles-link-6"
        href="/viewType/nightLife"
        onClick={[Function]}
      >
        <h5
          className="MuiTypography-root MuiTypography-h5"
        >
          Nightlife
        </h5>
      </a>
    </div>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item"
  >
    <div
      className="MuiGrid-root makeStyles-root-8 MuiGrid-container MuiGrid-direction-xs-column"
    >
      <div
        className="MuiGrid-root MuiGrid-item"
      >
        <h4
          className="MuiTypography-root MuiTypography-h4"
        >
          Collections
        </h4>
      </div>
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-item"
      >
        <div
          className="MuiGrid-root makeStyles-subHeading-9 MuiGrid-item MuiGrid-grid-lg-true"
        >
          <h6
            className="MuiTypography-root MuiTypography-subtitle1"
          >
            Explore curated lists of top restaurants, cafes, pubs, and bars in Hyderabad, based on trends
          </h6>
        </div>
        <div
          className="MuiGrid-root makeStyles-link-10 MuiGrid-container MuiGrid-item MuiGrid-align-items-xs-flex-end MuiGrid-justify-xs-flex-end MuiGrid-grid-lg-3"
        >
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <h6
              className="MuiTypography-root makeStyles-link-7 MuiTypography-subtitle1"
              mode="link"
            >
              All Collections in Hyderabad
            </h6>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M10 17l5-5-5-5v10z"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item"
      >
        <div
          className="MuiGrid-root makeStyles-root-11 MuiGrid-container MuiGrid-spacing-xs-4"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3"
          >
            <a
              href="/collections"
              onClick={[Function]}
              style={
                Object {
                  "textDecoration": "none",
                }
              }
            >
              <div
                className="MuiGrid-root makeStyles-root-12 MuiGrid-container MuiGrid-align-items-xs-flex-end"
                data-testid="main"
                style={
                  Object {
                    "backgroundImage": "linear-gradient(rgba(0,0,0, 0.1), rgb(0, 0, 0, .6)), url(collection1.jpg)",
                  }
                }
              >
                <div
                  className="MuiGrid-root makeStyles-content-14 MuiGrid-item"
                >
                  <h6
                    className="MuiTypography-root MuiTypography-h6"
                    data-testid="special"
                  >
                    New Year's Specials
                  </h6>
                  <h6
                    className="MuiTypography-root MuiTypography-subtitle2"
                    data-testid="place"
                  >
                    12
                     Places
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root makeStyles-arrowIcon-15 MuiSvgIcon-fontSizeSmall"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M10 17l5-5-5-5v10z"
                      />
                    </svg>
                  </h6>
                </div>
              </div>
            </a>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3"
          >
            <a
              href="/collections"
              onClick={[Function]}
              style={
                Object {
                  "textDecoration": "none",
                }
              }
            >
              <div
                className="MuiGrid-root makeStyles-root-12 MuiGrid-container MuiGrid-align-items-xs-flex-end"
                data-testid="main"
                style={
                  Object {
                    "backgroundImage": "linear-gradient(rgba(0,0,0, 0.1), rgb(0, 0, 0, .6)), url(collection2.jpg)",
                  }
                }
              >
                <div
                  className="MuiGrid-root makeStyles-content-14 MuiGrid-item"
                >
                  <h6
                    className="MuiTypography-root MuiTypography-h6"
                    data-testid="special"
                  >
                    Trending This Week
                  </h6>
                  <h6
                    className="MuiTypography-root MuiTypography-subtitle2"
                    data-testid="place"
                  >
                    31
                     Places
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root makeStyles-arrowIcon-15 MuiSvgIcon-fontSizeSmall"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M10 17l5-5-5-5v10z"
                      />
                    </svg>
                  </h6>
                </div>
              </div>
            </a>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3"
          >
            <a
              href="/collections"
              onClick={[Function]}
              style={
                Object {
                  "textDecoration": "none",
                }
              }
            >
              <div
                className="MuiGrid-root makeStyles-root-12 MuiGrid-container MuiGrid-align-items-xs-flex-end"
                data-testid="main"
                style={
                  Object {
                    "backgroundImage": "linear-gradient(rgba(0,0,0, 0.1), rgb(0, 0, 0, .6)), url(collection3.jpg)",
                  }
                }
              >
                <div
                  className="MuiGrid-root makeStyles-content-14 MuiGrid-item"
                >
                  <h6
                    className="MuiTypography-root MuiTypography-h6"
                    data-testid="special"
                  >
                    Hyderabad's Hottest
                  </h6>
                  <h6
                    className="MuiTypography-root MuiTypography-subtitle2"
                    data-testid="place"
                  >
                    111
                     Places
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root makeStyles-arrowIcon-15 MuiSvgIcon-fontSizeSmall"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M10 17l5-5-5-5v10z"
                      />
                    </svg>
                  </h6>
                </div>
              </div>
            </a>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-3"
          >
            <a
              href="/collections"
              onClick={[Function]}
              style={
                Object {
                  "textDecoration": "none",
                }
              }
            >
              <div
                className="MuiGrid-root makeStyles-root-12 MuiGrid-container MuiGrid-align-items-xs-flex-end"
                data-testid="main"
                style={
                  Object {
                    "backgroundImage": "linear-gradient(rgba(0,0,0, 0.1), rgb(0, 0, 0, .6)), url(collection4.jpg)",
                  }
                }
              >
                <div
                  className="MuiGrid-root makeStyles-content-14 MuiGrid-item"
                >
                  <h6
                    className="MuiTypography-root MuiTypography-h6"
                    data-testid="special"
                  >
                    Work Friendly Places!
                  </h6>
                  <h6
                    className="MuiTypography-root MuiTypography-subtitle2"
                    data-testid="place"
                  >
                    221
                     Places
                    <svg
                      aria-hidden={true}
                      className="MuiSvgIcon-root makeStyles-arrowIcon-15 MuiSvgIcon-fontSizeSmall"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M10 17l5-5-5-5v10z"
                      />
                    </svg>
                  </h6>
                </div>
              </div>
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item"
  >
    <h4
      className="MuiTypography-root MuiTypography-h4"
    >
      Nightlife
       Restaurants in Hyderabad
    </h4>
  </div>
</div>
`;
